speed-dreams/source-2.2.3/data/database_connection_settings.txt

```diff
@@ -2,4 +2,5 @@
 3306
 root
 PASSWORD
-sda_schema
\ No newline at end of file
+sda_schema
```

Comment:
is this supposed to be in here, if it shouldn't we need to change the password of the server

Reply:
that is the database name, should be in there

Reply:
This is also for the local database
the settingsfile for the remote is different
but not on github